### PR TITLE
Update tutorial part 1 to reflect changes to add site flow

### PR DESCRIPTION
**Pull request by aghreed**
## Description

Now that we have enabled the new add site experience for all users, it follows that we should update the instructions in the tutorial that reference that flow.

## Related Issues

[sc-42765](https://app.shortcut.com/gatsbyjs/story/42765/update-tutorial-references-to-add-site-flow-to-be-accurate-of-gercel-experiment)

## Testing

Many thanks to @rmatambo8 for discovering the indentation issue here that was plaguing me! All looks good to me now locally!

Screen Shot 2022-01-11 at 4 16 58 PM

### CODE REVIEW

**Review by rmatambo8 — APPROVED** [review]
It's amazing how many files needed changing for this.

LGTM!

File: docs/docs/tutorial/part-1/index.mdx [hunk 1]
```diff
@@ -254,47 +271,37 @@ To connect your code on GitHub to your Gatsby Cloud account, do the following:
...
 
-   ![The "Git provider" tab of the "Add a site" screen. It lists two options: GitHub or GitLab.](./03-select-a-git-provider.png)
-
-1. If this is your first time connecting GitHub to Gatsby Cloud, you'll need to give Gatsby Cloud permission to access your GitHub account. When asked, **"Where should we host this new project?"** click the **"Select destination"** button.
+1. If this is your first time connecting GitHub to Gatsby Cloud, you'll need to give Gatsby Cloud permission to access your GitHub account.
 
   <Announcement style={{marginBottom: "1.5rem"}}>
 
-  **Note:** If you are part of more than one GitHub organization, you will need to first select the organization with which the repository resides at this step before selecting the repository itself.
+    **Note:** If you are part of more than one GitHub organization, you will need to first select the organization with which the repository resides at this step before selecting the repository itself.
```
Comment (jxnblk) [inline docs/docs/tutorial/part-1/index.mdx:282]: FWIW, this indentation looks suspect to me – could also be related to the Announcement component being using inside the parent `ol`

Comment (aghreed) [reply docs/docs/tutorial/part-1/index.mdx:282]: With the latest commit on this branch by Rodney, I'm seeing everything work as expected. He is my personal hero today ❤️

**Review by rmatambo8 — APPROVED** [review]
:shipit:

**Pull request merged**